diff --git a/dist/node.js b/dist/node.js
index 513168c89e387668a79b6beb109587b3eb41a7d0..9047683d2efecd1bc8a413d259be42888d0b3ac1 100644
--- a/dist/node.js
+++ b/dist/node.js
@@ -7,7 +7,9 @@ let converter;
         converter = require('bindings')('bigint_buffer');
     }
     catch (e) {
-        console.warn('bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)');
+        if (process.env.WARN_ABOUT_BIGING_BUFFER) {
+            console.warn('bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)');
+        }
     }
 }
 /**
